bors.toml

```diff
@@ -1,3 +0,0 @@
-status = [
-  "continuous-integration/travis-ci/push",
-]
```

Comment:
Why'd you kill bors? 😮

Reply:
Well that is a mistake, sorry :) `.toml` 😬